src/pconn.cc

```diff
@@ -348,36 +350,44 @@ PconnPool::key(const Comm::ConnectionPointer &destLink, const char *domain)
 }
 
 void
-PconnPool::dumpHist(StoreEntry * e) const
+PconnPool::dumpHist(std::ostream &yaml) const
 {
-    storeAppendPrintf(e,
-                      "%s persistent connection counts:\n"
-                      "\n"
-                      "\t Requests\t Connection Count\n"
-                      "\t --------\t ----------------\n",
-                      descr);
+    AtMostOnce explanation(
```

Comment:
The primary purpose of this variable is (not to explain the meaning of histogram columns but) to provide a required heading for the histogram.

```suggestion
    AtMostOnce heading(
```